File: day034/homework/code/codewars6.py
# https://www.codewars.com/kata/546e2562b03326a88e000020/train/python


def square_digits(num):
    digits = str(num)
    squared_digits = [str(int(d)**2) for d in digits]
    result_str = ''.join(squared_digits)
    return int(result_str)

File: day068/homework/code/codewars6.js
// https://www.codewars.com/kata/54e6533c92449cc251001667/train/python

def unique_in_order(sequence):
    result = []
    prev = None
    for item in sequence:
        if item != prev:
            result.append(item)
            prev = item
    return result
